src/lib/Libutil/pbs_aes_encrypt.c

```diff
@@ -19,7 +19,7 @@ extern unsigned char pbs_aes_iv[];
 #define CIPHER_CONTEXT_CLEAN(v) EVP_CIPHER_CTX_cleanup(v)
 #else
 #define CIPHER_CONTEXT_INIT(v) v = EVP_CIPHER_CTX_new(); if (!v) return -1;
```

Comment:
Just a nit... the definition of CIPHER_CONTEXT_INIT is followed by a semicolon when it doesn't need to be.